cmd/config_validation_test.go

```diff
@@ -641,6 +641,40 @@ func TestValidateConfigFile_EnableHNSConfigSuccessful(t *testing.T) {
 	}
 }
 
+func TestValidateConfigFile_EnableAtomicRenameObjectConfigSuccessful(t *testing.T) {
```

Comment:
Please remove this test. It's not needed. Please refer to this on what tests to write - go/gcs-fuse-coding-guidelines#adding-new-param-in-gcsfuse